auth-api/src/auth_api/utils/base_enum.py

```diff
@@ -0,0 +1,41 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds all of the basic data about a business."""
+
+from enum import Enum, EnumMeta
+
+
+class BaseMeta(EnumMeta):
+    """Meta class for the enum."""
+
+    def __contains__(self, other):  # pylint: disable=C0203
+        """Return True if 'in' the Enum."""
+        try:
+            self(other)  # pylint: disable=no-value-for-parameter
+        except ValueError:
+            return False
+        else:
+            return True
+
+
+class BaseEnum(Enum, metaclass=BaseMeta):
```

Comment:
removed `str` here because it was converting everything to strings and breaking a bunch of tests